PVC/Mesh_voxelisation/COMPUTE_mesh_normals.m

```diff
@@ -0,0 +1,205 @@
+function [coordNORMALS,varargout] = COMPUTE_mesh_normals(meshdataIN,invertYN)
+% COMPUTE_mesh_normals  Calculate the normals for each facet of a triangular mesh
+%==========================================================================
+% AUTHOR        Adam H. Aitkenhead

```

Comment:
@gmangeat did you check if we have the permission to add in our repos with MIT license?